src/Adapters/Symfony/EventListener/ErrorListener.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * This file is part of Collision.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace NunoMaduro\Collision\Adapters\Symfony\EventListener;
+
+use Whoops\Exception\Inspector;
+use NunoMaduro\Collision\Provider;
+use Symfony\Component\Console\Event\ConsoleErrorEvent;
+
+/**
+ * This is an Collision Symfony Adapter Error Listener implementation.
+ *
+ * @author Nuno Maduro <enunomaduro@gmail.com>
+ */
+class ErrorListener
+{
+    /**
+     * Retreives error from the provided event
+     * and ouputs details of that error to the event output.
+     *
+     * @param \Symfony\Component\Console\Event\ConsoleErrorEvent $event
+     */
+    public function onConsoleError(ConsoleErrorEvent $event)
```

Comment:
I don't know if it's intended, but with the following code you would end up with a new instance of the handle (which is registered each time) on each console error. This might not be a big deal most of the time since I suspect most console errors result in the process termination, but that might not necessarily be the case e.g. during the tests which could lead to a memory leak (it's an example, I didn't check if that could result in one).

Reply:
I didn't get your point. Do you mind of rephrase it? This class is just a listener that is hearing for events of the type `Symfony\Component\Console\ConsoleEvents::ERROR`. I don't know symfony that much, but I believe that this class is only instantiated if an error occurs.

Reply:
The class `ErrorListener` is instantiated only once yes, but the handler generated by:

```php
        $handler = (new Provider())->register()
            ->getHandler()
            ->setOutput($event->getOutput());
```

is created each time an event of the type `Symfony\Component\Console\ConsoleEvents::ERROR` is triggered